hunts/src/HuntViewMain.js

```diff
@@ -138,7 +145,41 @@ export const HuntViewMain = (props) => {
       >
         {alert.text}
       </Alert>
-      <h1>{hunt.name} - All Puzzles</h1>
+      <div
+        style={{
+          display: "flex",
+          justifyContent: "space-between",
+          padding: "2px",
+          alignItems: "center",
+        }}
+      >
+        <h1>{hunt.name} - All Puzzles</h1>
+        <a href={"stats"}>Hunt Statistics</a>
+      </div>
+      <Container fluid>
+        <Row className="text-center font-weight-bold small">
+          <Col xs={1} className="text-nowrap">
+            Metas Solved
+          </Col>
+          <Col xs={1}>Solved</Col>
+          <Col xs={1}>Unsolved</Col>
+          <Col xs={1}>Unlocked</Col>
+        </Row>
+        <Row className="text-center font-weight-bold">
+          <Col xs={1} className="text-primary">
+            {hunt.num_metas_solved}
+          </Col>
+          <Col xs={1} className="text-success">
+            {hunt.num_solved}
+          </Col>
+          <Col xs={1} className="text-danger">
+            {hunt.num_unsolved}
+          </Col>
+          <Col xs={1} className="text-secondary">
+            {hunt.num_unlocked}
+          </Col>
+        </Row>
+      </Container>
```

Comment:
Maybe we can split this out into another component, this file/component is already kinda busy (my fault)